evadb/constants.py

```diff
@@ -21,3 +21,4 @@
 IFRAMES = "IFRAMES"
 AUDIORATE = "AUDIORATE"
 DEFAULT_FUNCTION_EXPRESSION_COST = 100
+LLM_FUNCTIONS = ["chatgpt", "completion"]
```

Comment:
Are we not adding the LLM operator to the parser? So, the allowed LLM names are restricted here?